pkg/scheduler/core/generic_scheduler_test.go

```diff
@@ -1198,6 +1206,14 @@ func TestPickOneNodeForPreemption(t *testing.T) {
 	}
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
+			now := metav1.Now()
```

Comment:
These code are same as line#980, perhaps we could create a function for it.

Reply:
done